src/js/components/Select/SelectContainer.js

```diff
@@ -390,7 +391,12 @@ class SelectContainer extends Component {
             overflow="auto"
           >
             {options.length > 0 ? (
-              <InfiniteScroll items={options} step={theme.select.step} replace>
+              <InfiniteScroll
+                items={options}
+                step={theme.select.step}
+                onMore={onMore}
+                replace
```

Comment:
maybe we should add replace to props as well?

Reply:
Would this work? I'm not sure how to deal with booleans.
`<InfiniteScroll ....................
      {this.prop.replace} >`

Reply:
just like you did with onMore. `replace={replace}`. and destructure it before. you might want to set  the defaultProps as well to true.